### PR TITLE
unzip: refresh page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** latest

### CODE REVIEW

File: pages/common/unzip.md [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
+`unzip -c {{path/to/archive1 path/to/archive2 ...}}`
 
-- Extract the contents of the file(s) to stdout alongside the extracted file names:
+- Extract files/directories from archives created on Windows containing files/directories with non-ASCII (e.g. Chinese or Japanese characters) names:
 
-`unzip -c {{file.zip}}`
+`unzip -O {{gbk}} {{path/to/archive1 path/to/archive2 ...}}`
 
-- Extract a zip file created on Windows, containing files with non-ASCII (e.g. Chinese or Japanese characters) filenames:
+- Print a specific archive contents:
```
Comment (mfrw) [inline pages/common/unzip.md:23]: How about ?
```suggestion
- Print contents of a specific archive :
```

**Review by mfrw — APPROVED** [review]
LGTM with a teeny cosmetic nitpic

File: pages/common/unzip.md [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
+`unzip -c {{path/to/archive1 path/to/archive2 ...}}`
 
-- Extract the contents of the file(s) to stdout alongside the extracted file names:
+- Extract files/directories from archives created on Windows containing files/directories with non-ASCII (e.g. Chinese or Japanese characters) names:
 
-`unzip -c {{file.zip}}`
+`unzip -O {{gbk}} {{path/to/archive1 path/to/archive2 ...}}`
 
-- Extract a zip file created on Windows, containing files with non-ASCII (e.g. Chinese or Japanese characters) filenames:
+- Print contents of a specific archive:
```
Comment (einverne) [inline pages/common/unzip.md:23]:
```suggestion
- List the contents of a specific archive without extracting:
```

Still think the origin version is better to understand.

**Review by einverne — DISMISSED** [review]

File: pages/common/unzip.md (hunk 2 above)
Comment (EmilySeville7cfg) [reply pages/common/unzip.md:23]: What about using `list` verb just for cases when we are printing `\n` separated items and `print` for the others?

Comment (CleanMachine1) [reply pages/common/unzip.md:23]:
```suggestion
- List the contents of a specific archive without extracting them:
```

File: pages/common/unzip.md [hunk 3]
```diff
@@ -1,24 +1,25 @@
 # unzip
 
-> Extract compressed files in a ZIP archive.
+> Extract compressed files/directories from zip archives.
```
Comment (marchersimon) [inline pages/common/unzip.md:3]: It's either ``` `.zip` ``` or `ZIP`. (source: Wikipedia)

File: pages/common/unzip.md [hunk 4]
```diff
@@ -1,24 +1,25 @@
...
+> See also: `zip`.
 > More information: <https://manned.org/unzip>.
 
-- Extract zip file(s) (for multiple files, separate file paths by spaces):
+- Extract files/directories from specific archives:
```
Comment (marchersimon) [inline pages/common/unzip.md:7]:
```suggestion
- Extract all files/directories from specific archives into the current directory:
```
Or maybe even
```suggestion
- Extract specific archives to the current directory:
```

File: pages/common/unzip.md [hunk 5]
```diff
@@ -1,24 +1,25 @@
...
 
-`unzip {{file(s)}}`
+`unzip {{path/to/archive1 path/to/archive2 ...}}`
```
Comment (marchersimon) [inline pages/common/unzip.md:9]: I totally get your point, that we should use more paths to show that the command can handle multiple files at once. On the other side, tough, I think this just makes the examples longer and a tiny little harder to understand. In the end we should write the examples in a way that is most useful to the user. And in 99% of the cases you want to unzip just one single archive.

File: pages/common/unzip.md [hunk 6]
```diff
@@ -1,24 +1,25 @@
...
+`unzip {{path/to/archive1 path/to/archive2 ...}} -d {{path/to/output}}`
 
-- List the contents of a zip file without extracting:
+- Extract files/directories from archives to stdout alongside extracted file/directory names:
 
-`unzip -l {{file.zip}}`
+`unzip -c {{path/to/archive1 path/to/archive2 ...}}`
 
-- Extract the contents of the file(s) to stdout alongside the extracted file names:
+- Extract files/directories from archives created on Windows containing files/directories with non-ASCII (e.g. Chinese or Japanese characters) names:
```
Comment (marchersimon) [inline pages/common/unzip.md:19]: I think we can shorten this a bit.
```suggestion
- Extract files/directories from archives created on Windows containing non-ASCII file names:
```

File: pages/common/unzip.md [hunk 7]
```diff
@@ -1,24 +1,25 @@
...
+`unzip {{path/to/archive1 path/to/archive2 ...}}`
 
-- Extract zip files(s) to given path:
+- Extract files/directories from archives to a specific path:
 
-`unzip {{compressed_file(s)}} -d {{path/to/put/extracted_file(s)}}`
+`unzip {{path/to/archive1 path/to/archive2 ...}} -d {{path/to/output}}`
 
-- List the contents of a zip file without extracting:
+- Extract files/directories from archives to stdout alongside extracted file/directory names:
```
Comment (marchersimon) [inline pages/common/unzip.md:15]: This sounds a little like without this option, the command doesn't print the file names to stdout. How about
```suggestion
- Extract files/directories from archives to `stdout`:
```

File: pages/common/unzip.md [hunk 8]
```diff
@@ -1,24 +1,25 @@
...
-> Extract compressed files in a ZIP archive.
+> Extract compressed files/directories from .zip archives.
+> See also: `zip`.
 > More information: <https://manned.org/unzip>.
 
-- Extract zip file(s) (for multiple files, separate file paths by spaces):
+- Extract all files/directories from specific archives into the current directory:
 
-`unzip {{file(s)}}`
+`unzip {{path/to/archive1 path/to/archive2 ...}}`
```
Comment (pixelcmtd) [inline pages/common/unzip.md:9]:
```suggestion
`unzip {{path/to/archive1.zip path/to/archive2.zip ...}}`
```

File: pages/common/unzip.md [hunk 9]
```diff
@@ -1,24 +1,25 @@
...
 
-- Extract zip files(s) to given path:
+- Extract files/directories from archives to a specific path:
 
-`unzip {{compressed_file(s)}} -d {{path/to/put/extracted_file(s)}}`
+`unzip {{path/to/archive1 path/to/archive2 ...}} -d {{path/to/output}}`
```
Comment (pixelcmtd) [inline pages/common/unzip.md:13]:
```suggestion
`unzip {{path/to/archive1.zip path/to/archive2.zip ...}} -d {{path/to/output}}`
```

File: pages/common/unzip.md [hunk 10]
```diff
@@ -1,24 +1,25 @@
...
 
-- List the contents of a zip file without extracting:
+- Extract files/directories from archives to `stdout`:
 
-`unzip -l {{file.zip}}`
+`unzip -c {{path/to/archive1 path/to/archive2 ...}}`
```
Comment (pixelcmtd) [inline pages/common/unzip.md:17]:
```suggestion
`unzip -c {{path/to/archive1.zip path/to/archive2.zip ...}}`
```

File: pages/common/unzip.md [hunk 11]
```diff
@@ -1,24 +1,25 @@
...
 
-- Extract the contents of the file(s) to stdout alongside the extracted file names:
+- Extract files/directories from archives created on Windows containing non-ASCII file names:
 
-`unzip -c {{file.zip}}`
+`unzip -O {{gbk}} {{path/to/archive1 path/to/archive2 ...}}`
```
Comment (pixelcmtd) [inline pages/common/unzip.md:21]:
```suggestion
`unzip -O {{gbk}} {{path/to/archive1.zip path/to/archive2.zip ...}}`
```

File: pages/common/unzip.md [hunk 12]
```diff
@@ -1,24 +1,25 @@
 # unzip
 
-> Extract compressed files in a ZIP archive.
+> Extract compressed files/directories from .zip archives.
```
Comment (pixelcmtd) [inline pages/common/unzip.md:3]: Files in a ZIP archive aren't necessarily compressed (and directories can't be).

```suggestion
> Extract files/directories from ZIP archives.
```

**Review by kbdharun — APPROVED** [review]

**Pull request merged**